xml/ha_storage_protection.xml

```diff
@@ -1373,6 +1373,421 @@ Illegal request, Invalid opcode</screen>
   </sect2>
  </sect1>
 
+ <sect1 xml:id="sec-ha-storage-protect-sbd-redeploy">
+  <title>Changing SBD configuration</title>
+  <para>
+    You might need to change the cluster's SBD configuration for various reasons. For example:
+  </para>
+  <itemizedlist>
+   <listitem>
+     <para>
+       Changing disk-based SBD to diskless SBD
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Changing diskless SBD to disk-based SBD
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Replacing an SBD device with a new device
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Changing timeout values and other settings
+     </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+    You can use &crmsh; to change the SBD configuration. This method uses
+    &crmsh;'s default settings, including timeout values.
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        <xref linkend="pro-ha-storage-protect-sbd-redeploy-disk-based-crmsh"/>
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        <xref linkend="pro-ha-storage-protect-sbd-redeploy-diskless-crmsh"/>.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <para>
+    If you need to change any settings,
+    or if you use custom settings and need to retain them when replacing a device, you must
+    manually edit the SBD configuration.
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        <xref linkend="pro-ha-storage-protect-sbd-redeploy-disk-based-manual"/>
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        <xref linkend="pro-ha-storage-protect-sbd-redeploy-diskless-manual"/>
+      </para>
+    </listitem>
+  </itemizedlist>
+
+  <procedure xml:id="pro-ha-storage-protect-sbd-redeploy-disk-based-crmsh">
+    <title>Redeploying disk-based SBD with &crmsh;</title>
+    <para>
+      Use this procedure to change diskless SBD to disk-based SBD, or to replace an existing
+      SBD device with a new device.
+    </para>
+    <step>
+      <para>
+        Put the cluster into maintenance mode:
+      </para>
+<screen>&prompt.root;<command>crm maintenance on</command></screen>
+      <para>
+        In this state, the cluster stops monitoring all resources, so the services managed
+        by the resources can continue to run even if the cluster services stop.
```

Comment:
Would it make sense to point users to the following sections:

https://susedoc.github.io/doc-sleha/main/single-html/SLE-HA-administration/#sec-ha-maint-mode-cluster
https://susedoc.github.io/doc-sleha/main/single-html/SLE-HA-administration/#sec-ha-maint-cluster-stop

of course depending on which suits their case better.